.travis.yml

```diff
@@ -8,6 +8,7 @@ python:
   - 3.4
   - 2.7
   - pypy
+  - nightly
```

Comment:
do we really need it? we aren't using cutting edge feature of python

Reply:
testing on 3.7, it should be fine

Reply:
alright, 3.7 in.